pkg/services/sqlstore/migrator/postgres_dialect.go

```diff
@@ -155,3 +155,15 @@ func (db *Postgres) IsUniqueConstraintViolation(err error) bool {
 func (db *Postgres) IsDeadlock(err error) bool {
 	return db.isThisError(err, "40P01")
 }
+
+func (db *Postgres) PostInsertId(table string, sess *xorm.Session) error {
+	if table != "org" {
+		return nil
+	}
+
+	// sync primary key sequence of org table
+	if _, err := sess.Exec("SELECT setval('org_id_seq', (SELECT max(id)+1 FROM org));"); err != nil {
+		return fmt.Errorf("Failed to sync primary key for org table")
```

Comment:
This seems to follow the error pattern in the file, but I wonder:
 1. Why not return the error if an error?
 2. Regardless of what is done with the error, should the contents of sess.Exec's err be wrapped into the message so the reason is logged?

Reply:
good points, I will fix it!

Reply:
Yeah, wrapping the error sounds like the thing to do for me as well :)